### DISCUSSION TITLE
Thursday Jul 28 Evening Group Project Meetup 7pm ET Zoom?

### DISCUSSION THREAD

**Discussion by matty10s** [matty10s/group3-project#14] [Announcements]
Would both of you be available for a quick Zoom meeting at 7pm ET this evening?

I have a doctors appt from 2:30PM - 4PM Mountain (4:30pm-6pm ET) but I am available anytime before or after.

**Comment by shannonkuhn92** [comment]
I'll be there

**Reply by shannonkuhn92** [reply]
Just use the usual zoom link from Canvas

**Comment by tay-baum** [comment]
I should be able to also.

**Comment by shannonkuhn92** [comment]
I THINK I FIXED IT. PLEASE REACH OUT DIRECTLY IF FOR SOME REASON I AM CURSED AND HAVE RUINED EVERYTHING ONCE AGAIN.   

(ノ°-°）ノ︵ ┻━┻
